src/radical/pilot/agent/rm/fork.py

```diff
@@ -49,8 +49,7 @@ def _configure(self):
             self.cores_per_node = 1
 
         self.node_list  = list()
-        requested_nodes = int(math.ceil(float(self.requested_cores) /
-                                        float(self.cores_per_node ) ) )
+        requested_nodes = int(math.ceil(float(self.requested_cores) / float(self.cores_per_node ) ) )
         for i in range(requested_nodes):
```

Comment:
why?  This is now longer than 80?

Reply:
I think we instruct flake8 not to look for 80 characters. I can check

Reply:
I would keep the 80 characters rule. Even if all our monitors can accomodate longer lines these days, the limit helps to notice when the code becomes unreasonable 'deep'. Would you agree?

Reply:
:+1:   There are always exception one can consider - this shouldn't be one.